### PR TITLE
Fixes major memory leak in component renderer

### PR DESCRIPTION
**Pull request by wwwillchen**
If you navigate to the same pages over and over again, you can measure an significant increase in JS memory (heap snapshots). The main contributor was that we weren't cleaning up the embedded view refs.

### CODE REVIEW

File: mesop/version.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """Contains the version string."""
 
-VERSION = "0.12.9"
+VERSION = "0.12.10beta5"
```
Comment (richard-to) [inline mesop/version.py:3]: Is this change necessary?

**Review by richard-to — APPROVED** [review]
LGTM - Great find.

File: mesop/version.py (hunk 1 above)
Comment (wwwillchen) [reply mesop/version.py:3]: let me take it out. I'll cut a new release soon
